app/src/main/java/com/duckduckgo/app/trackerdetection/blocklist/BlockList.kt

```diff
@@ -0,0 +1,139 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.trackerdetection.blocklist
+
+import com.duckduckgo.anvil.annotations.ContributesRemoteFeature
+import com.duckduckgo.app.trackerdetection.blocklist.BlockList.Companion.EXPERIMENT_PREFIX
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.feature.toggles.api.ConversionWindow
+import com.duckduckgo.feature.toggles.api.FeatureTogglesInventory
+import com.duckduckgo.feature.toggles.api.MetricsPixel
+import com.duckduckgo.feature.toggles.api.MetricsPixelPlugin
+import com.duckduckgo.feature.toggles.api.Toggle
+import com.duckduckgo.feature.toggles.api.Toggle.State.CohortName
+import com.squareup.anvil.annotations.ContributesMultibinding
+import javax.inject.Inject
+
+@ContributesRemoteFeature(
+    scope = AppScope::class,
+    featureName = "blockList",
+)
+interface BlockList {
+    @Toggle.DefaultValue(false)
+    fun self(): Toggle
+
+    @Toggle.DefaultValue(false)
+    fun tdsNextExperimentBaseline(): Toggle
+
+    @Toggle.DefaultValue(false)
+    fun tdsNextExperimentBaselineBackup(): Toggle
+
+    @Toggle.DefaultValue(false)
+    fun tdsNextExperimentNov24(): Toggle
+
+    @Toggle.DefaultValue(false)
+    fun tdsNextExperimentDec24(): Toggle
+
+    @Toggle.DefaultValue(false)
+    fun tdsNextExperimentJan25(): Toggle
+
+    @Toggle.DefaultValue(false)
+    fun tdsNextExperimentFeb25(): Toggle
+
+    @Toggle.DefaultValue(false)
+    fun tdsNextExperimentMar25(): Toggle
+
+    enum class Cohorts(override val cohortName: String) : CohortName {
+        CONTROL("control"),
+        TREATMENT("treatment"),
+    }
+
+    companion object {
+        const val EXPERIMENT_PREFIX = "tds"
+        const val TREATMENT_URL = "treatmentUrl"
+        const val CONTROL_URL = "controlUrl"
+        const val NEXT_URL = "nextUrl"
+    }
+}
+
+@ContributesMultibinding(AppScope::class)
+class BlockListPixelsPlugin @Inject constructor(private val inventory: FeatureTogglesInventory) : MetricsPixelPlugin {
+
+    override suspend fun getMetrics(): List<MetricsPixel> {
+        val activeToggle = inventory.activeTdsFlag() ?: return emptyList()
+
+        return listOf(
+            MetricsPixel(
+                metric = "2xRefresh",
+                value = "1",
+                toggle = activeToggle,
+                conversionWindow = listOf(
+                    ConversionWindow(lowerWindow = 0, upperWindow = 0),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 1),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 2),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 3),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 4),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 5),
```

Comment:
Nit: These could be made more concise: 
`conversionWindow = (0..5).map { ConversionWindow(lowerWindow = 0, upperWindow = it) }`